apps/wallet/src/components/external-canisters/CanisterIcSettingsForm.vue

```diff
@@ -0,0 +1,358 @@
+<template>
+  <VForm ref="form" @submit.prevent="submit">
+    <VContainer class="px-0 py-2">
+      <VRow>
+        <VCol cols="12" class="pb-0">
+          <CanisterIdField
+            v-if="props.display.canisterId || !model.canisterId"
+            v-model="model.canisterId"
+            name="canister_id"
+            density="comfortable"
+            :readonly="props.readonly"
+            required
+          />
+        </VCol>
+        <VCol cols="12" class="px-2 pt-0">
+          <VAlert v-if="hasNonOrbitControllers" type="warning" variant="tonal" density="compact">
+            {{ $t('external_canisters.non_orbit_controllers_warning') }}
+          </VAlert>
+        </VCol>
+        <VCol cols="12" class="py-0">
+          <p class="text-h6 mb-2">
+            {{ $t('external_canisters.native_settings.controllers') }}
+            <VChip rounded size="small" variant="tonal">
+              {{ model.controllers?.length || 0 }}
+            </VChip>
+          </p>
+          <VDivider />
+        </VCol>
+        <VCol cols="12" class="d-flex flex-column pt-0">
+          <VTable v-if="model.controllers && model.controllers.length" density="compact" hover>
+            <tbody>
+              <tr v-for="(controller, idx) in model.controllers" :key="idx">
+                <td class="pa-2 w-100">
+                  <span v-if="station.canisterId === controller.toText()">
+                    ({{ $t('app.name') }})
```

Comment:
This displays Orbit Wallet, but shouldn't this be Orbit Station? Since this is displayed if the station is the controller.

Slight nit: can we swap the order, eg. `ryjl3-tyaaa-aaaaa-aaaba-cai (Orbit Station)`